submissions/peterIyiola/html/productPage-1.html

```diff
@@ -6,9 +6,10 @@
    </head>
    <body>
       <div>
-         <a href="#">
+         
             <h1><b>Better Buys</b></h1>
-         </a>
+          <a href="C:\Users\HP\Desktop\Dufuna-CodeCamp20\submissions\peterIyiola\html\productPage-1.html">Product page</a><br><br>
```

Comment:
Well done! But this is not the best approach to link your pages. You need to use relative path, you this `[link](https://www.w3schools.com/Html/html_filepaths.asp)` for better understanding

Reply:
Okay